demos/taco-demo/src/App.tsx

```diff
@@ -89,7 +88,7 @@ export default function App() {
       provider,
       domain,
       encryptedMessage,
-      getPorterUri(domain),
+      undefined,
```

Comment:
We can get rid of this line, right?
```suggestion

```

Reply:
vs code shows error (and pnpm ci:lint) when I remove undefined, looks like it's stil using old interface with `porterUri?: string`. I'll keep looking

Reply:
We can only remove it if it's the last parameter in a function call - It will be implicitly set as `undefined`. Otherwise, we have to set it as `undefined` to preserve parameter ordering for further parameters.

Reply:
> We can only remove it if it's the last parameter in a function call - It will be implicitly set as `undefined`. Otherwise, we have to set it as `undefined` to preserve parameter ordering for further parameters.

Oh, got it.